mod/photos.php

```diff
@@ -196,7 +196,7 @@ function photos_post(App $a)
 		$album = hex2bin($a->argv[3]);
 
 		if ($album === L10n::t('Profile Photos') || $album === 'Contact Photos' || $album === L10n::t('Contact Photos')) {
-			goaway($_SESSION['photo_return']);
+			$a->internalRedirect($_SESSION['photo_return']);
```

Comment:
As far as I can see, `$_SESSION['photo_return']` is alwasy a relativ path, isn't it? Just to be sure ..

Reply:
OK I'm now sure (again), because I replaced `System::baseURL() . '/'. $_SESSION['photo_return']` at other places :-)